pkg/cloudprovider/providers/gce/gce_loadbalancer_utils_test.go

```diff
@@ -65,10 +67,24 @@ func DefaultTestClusterValues() TestClusterValues {
 	}
 }
 
-var fakeApiService *v1.Service
+var (
+	fakeApiService = &v1.Service{
```

Comment:
ApiService is very misleading. I thought this was referring to compute.Service for awhile.

Please rename all ApiService uses to something else. Maybe just fakeService or fakeK8sService

Reply:
Done. No more apiServices in the unit test.